### PR TITLE
Go Testing

### PR DESCRIPTION
**Pull request by jrg94**
Congrats on taking the first step to contributing to the [Sample Programs in Every Language][1] repository!

For simplicity, please make sure that your pull request includes one and only one sample program and language.

## Please Complete the Following

- [ ] I would **NOT** like credit for my contribution in the article (credit will be included otherwise)
- [ ] I plan to write the article for this code snippet
- [ ] I named the pull request using `<Sample Program> in <Language>` format
- [ ] I created/updated the language README
  - [ ] I added the sample program name to the README
  - [ ] I added fun facts (i.e. debut, developer, typing, etc.)
  - [ ] I added reference link(s) to the README
  - [ ] I added solution citations when necessary (see [plagiarism][2])
- [ ] I fixed #your-issue-number-here

## Notes

Feel free to put whatever you want here.

[1]: https://therenegadecoder.com/code/sample-programs-in-every-language/
[2]: https://github.com/TheRenegadeCoder/sample-programs/blob/master/CONTRIBUTING.md#plagiarism

### CODE REVIEW

File: archive/g/go/testinfo.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+folder:
+  extension: ".go"
+  naming: "hyphen"
+
+container:
+  image: "golang"
+  tag: "1"
```
Comment (auroq) [inline archive/g/go/testinfo.yml]: Go has opted to do versioning as x.y... where every change in y is a major release, so let's make the tag 1.12 to be safe.

Comment (jrg94) [reply archive/g/go/testinfo.yml]: Ah I see. The Version 1 image was updated 6 days, so I kind of lazily selected that for simplicity.

**Comment by jrg94** [comment]
Cool changes! I'm a fan. 👍

**Comment by auroq** [comment]
@jrg94 This is ready to be merged

**Pull request merged**